backend/inspirehep/search/facets.py

```diff
@@ -52,6 +55,24 @@ def inner(values):
     return inner
 
 
+def conferences_start_date_range_filter():
+    date_range_filter = range_filter("opening_date")
+
+    def inner(values):
+        value = values and values[0]
+
+        if value == "upcoming":
+            today = datetime.today().strftime("%Y-%m-%d")
+            return Range(**{"opening_date": {"gte": today}})
```

Comment:
we can use something more elegant https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-range-query.html#ranges-on-dates

Reply:
this makes it hard to test :/